apps/ryal_core/lib/ryal/web/models/payment_method.ex

```diff
@@ -50,14 +53,14 @@ defmodule Ryal.PaymentMethod do
   @payment_method_types ~w(credit_card)
 
   @doc """
-  You hand us some `data` and a `type` and we associate a payment method to a
-  user.
+  You hand us a `proxy` of data and a `type` and we associate a payment method
+  to a user.
 
   For an example on how to use this function, see the module description.
   """
   def changeset(struct, params \\ %{}) do
     struct
-    |> cast(set_module_type(params), @required_fields)
+    |> cast(set_module_type(params), @required_fields ++ [:ephemeral])
```

Comment:
A small thing but couldn't we just add this to `@required_fields` variable rather than appending to the list